microk8s-resources/actions/enable.cilium.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+if ! which helm 2>&1 >/dev/null; then
```

Comment:
I would prefer to have helm part of MicroK8s instead of asking the user to install it manually. It is a better experience plus it is less risky if we ensure we deploy and configure a helm that we know it works.

For now, this approach is fine. Adding helm could be handled in a separate PR.

Reply:
Our CI scripts need to install and configure helm so this passes testing. We run tests in lxc containers https://github.com/ubuntu/microk8s/blob/master/tests/test-distro.sh so the scripts for preparing these containers will need to be updated as well https://github.com/ubuntu/microk8s/tree/master/tests/lxc/install-deps.

Reply:
It might be easier to add a `microk8s.enable helm` that would provide a `microk8s.helm` command. Do you think you could provide that as part of this PR?

Reply:
I can take a stab at this.